lib/config/accountsConfig.js

```diff
@@ -0,0 +1,78 @@
+var mySubmitFunc = function(error, state){
+  if (!error) {
+    if (state === "signIn") {
+      // Successfully logged in
+      // ...
+    }
+    if (state === "signUp") {
+      // Successfully registered
+      // ...
+    }
+  }
+};
+
+AccountsTemplates.configure({
+    // Behaviour
+    confirmPassword: true,
+    enablePasswordChange: true,
+    forbidClientAccountCreation: false,
+    overrideLoginErrors: false,
+    sendVerificationEmail: false,
+    lowercaseUsername: false,
+
+    // Appearance
+    showAddRemoveServices: false,
+    showForgotPasswordLink: true,
+    showLabels: true,
+    showPlaceholders: true,
+    showResendVerificationEmailLink: false,
+
+    // Client-side Validation
+    continuousValidation: false,
+    negativeFeedback: false,
+    negativeValidation: true,
+    positiveValidation: true,
+    positiveFeedback: true,
+    showValidating: true,
+
+    // Privacy Policy and Terms of Use
+    privacyUrl: 'privacy',
+    termsUrl: 'terms-of-use',
+
+    // Redirects
+    homeRoutePath: '/',
+    redirectTimeout: 4000,
+
+    // Hooks
+    onSubmitHook: mySubmitFunc,
+    /*
+    onLogoutHook: myLogoutFunc,
+    preSignUpHook: myPreSubmitFunc,
+    */
+
+    // Texts
+    texts: {
+      button: {
+          signUp: "Register Now!"
+      },
+      socialSignUp: "Register",
+      socialIcons: {
+          "meteor-developer": "fa fa-rocket"
+      },
+      title: {
+          forgotPwd: "Recover Your Password"
+      },
+    },
+});
+
+/*
+AccountsTemplates.addFields([
```

Comment:
Do we need this? Can you implement what you're laying out here? If not we can remove it.